infrastructure/terraform/environments/global_aws/github_actions_iam.tf

```diff
@@ -128,4 +144,16 @@ resource "aws_iam_role_policy_attachment" "gha_attach_ecs" {
 resource "aws_iam_role_policy_attachment" "gha_attach_secrets" {
   role       = aws_iam_role.github_actions_role.name
   policy_arn = aws_iam_policy.github_actions_secrets.arn
-}
\ No newline at end of file
+}
+
+// TODO: create a new user for pullpreview and remove both user policy attachments below
+
+resource "aws_iam_user_policy_attachment" "gha_user_attach_lightsail" {
+  user       = aws_iam_user.github_actions.name
+  policy_arn = aws_iam_policy.lightsail.arn
+}
+
+resource "aws_iam_user_policy_attachment" "gha_user_attach_ecr" {
+  user       = aws_iam_user.github_actions.name
+  policy_arn = aws_iam_policy.ecr.arn
+}
```

Comment:
yeah, and ideally i think we should only allow read access to ECR for that user right?